packages/styles/scripts/autolayout.ts

```
@@ -63,44 +63,43 @@ export const DEFAULT_SPACING_MATRIX: DefaultSpacingMatrix = [
-  let mobileContent = '';
+  let scssContent = 'body:not(.is-tight) {\n'
+  let mobileContent = ''
 
   for (const entry of spacingMatrix) {
-    const [insertType, component, component2, spacingValue, mobileSpacingValue] = entry;
-    let selector = '';
+    const [insertType, component, component2, spacingValue, mobileSpacingValue] = entry
+    let selector = ''
 
     if (insertType === INSERT_SPACE_BETWEEN) {
       if (!component2 || typeof component2 === null) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)